File: Structural/Bridge/JavaScript/advancedRemote.js
// advancedRemote.js

const BasicRemote = require("./basicRemote");

class AdvancedRemote extends BasicRemote {
  mute() {
    this.device.setVolume(0);
    console.log("Device is muted.");
  }
}

module.exports = AdvancedRemote;

File: Structural/Bridge/CSharp/AdvancedRemote.cs
// AdvancedRemote.cs

using System;

public class AdvancedRemote : BasicRemote {
  public AdvancedRemote(IDevice device) : base(device) {}

  public void Mute() {
    device.SetVolume(0);
    Console.WriteLine("Device is muted.");
  }
}
